### PR TITLE
Fixes Issue with Megaseed Vendor not showing names of seed packets.

### PR DESCRIPTION
**Pull request by Derrick2Electricboogaloo**
## About The Pull Request

Fixes Issue #10950, increases seed packet quantity from 3 to 10.

## Why It's Good For The Game

Bug fix good.

## Changelog
:cl:
fix: Seed Vendors now show the proper seed packets.
/:cl:

### CODE REVIEW

**Comment by Derrick2Electricboogaloo** [comment]
If there is a better way for me to name the seeds in the vendor instead of copy and pasting name = "" let me know, so I can learn.

**Comment by Derrick2Electricboogaloo** [comment]
Hmmm, not sure why the vending types is showing as if I am adding the entire file as an addition. I only messed with Megaseed vendor.

**Comment by TiviPlus** [comment]
Because this is a webedit and what you did was copy the entire file into the root folder

**Comment by Derrick2Electricboogaloo** [comment]
> Because this is a webedit and what you did was copy the entire file into the root folder

DARN IT. Okay. Will try to fix. Source tree is just being so uncooperative.

**Comment by Derrick2Electricboogaloo** [comment]
Oh yep, did not think it just drops right into the root. Dammit.

**Comment by Derrick2Electricboogaloo** [comment]
Ready for speed merge now :)

File: code/modules/hydroponics/seeds.dm [hunk 1]
```diff
@@ -28,111 +28,143 @@
 
 
 /obj/item/seeds/poppyseed
+	name = "Poppy Seed"
```
Comment (TiviPlus) [inline code/modules/hydroponics/seeds.dm:31]: these are all improper nouns

Comment (Derrick2Electricboogaloo) [reply code/modules/hydroponics/seeds.dm:31]: Fucking how, how does making them lower case make or break them being improper nouns? I need an explanation.

Comment (Derrick2Electricboogaloo) [reply code/modules/hydroponics/seeds.dm:31]: And if anything they should be capitalized, because it is the name of a thing

**Comment by FosterGrime-Heymynameslouie** [comment]
can finally become a true botanist without having to play the lottery everytime I want to vend something or entirely draining the vendor and alt clicking, giving me a 50 50 chance on whether or not my game crashes


spedde mwhfeerd

**Comment by Derrick2Electricboogaloo** [comment]
> can finally become a true botanist without having to play the lottery everytime I want to vend something or entirely draining the vendor and alt clicking, giving me a 50 50 chance on whether or not my game crashes
> 
> spedde mwhfeerd

Absolutely. Just waiting on the merge.

**Pull request merged**